terraform/deployments/elasticsearch/main.tf

```diff
@@ -42,16 +42,8 @@ locals {
     data.tfe_outputs.vpc.nonsensitive_values.private_subnet_ids["elasticsearch_b"],
     data.tfe_outputs.vpc.nonsensitive_values.private_subnet_ids["elasticsearch_c"],
   ]
-  // master_user = "${var.service}-masteruser"
```

Comment:
Squash into the previous commit?

Reply:
I'll actually squash merge the whole thing, it doesn't need to be separate commits in this case